logic_tree_tools.py

```diff
@@ -289,6 +298,8 @@ def select_branch_sets_given_tectonic_region_type(
     if isinstance(tectonic_region_type_set, str):
         tectonic_region_type_set = [tectonic_region_type_set]
 
+    print()
```

Comment:
Random print?

Reply:
sorry, this was probably for a debugging break point